content/getting-started/javascript.textile

```diff
@@ -0,0 +1,129 @@
+---
+title: "Getting started: Pub/Sub and JavaScript"
+meta_description: "A getting started guide for Ably Pub/Sub that steps through some of the key features using JavaScript."
+meta_keywords: "Ably, realtime, quickstart, getting started, basics, pub/sub"
+---
+
+This guide will get you started with Ably Pub/Sub in JavaScript.
+
+It will take you through the following steps:
+
+* Complete the prerequisites before you begin.
+* Create a client and establish a realtime connection to Ably.
+* Attach to a channel and subscribe to its messages.
+* Publish a message to the channel for your client to receive.
+* Join and subscribe to the presence set of the channel.
+* Retrieve the messages you sent in the guide from history.
+
+h2(#prerequisites). Prerequisites
+
+* Sign up for an Ably account.
+* Create a new app, and create your first API key.
+** Your API key will need the @publish@, @subscribe@, @presence@ and @history@ capabilities.
+* Install the Ably CLI: @npm install -g &#64;ably/cli@
+* Run @ably login@ in the terminal and follow the prompts to provide an access token to the Ably CLI.
+* Set the default app to use with @ably apps switch@ and select an API key to use for operations with @ably auth keys switch@.
+
+* Install the Ably Pub/Sub JavaScript SDK in a new project within your IDE: @npm install ably@
+
+h2(#step-1). Step 1: Connect to Ably
+
+Clients establish a connection with Ably when they instantiate an SDK. This enables them to send and receive messages in realtime across channels.
+
+* Open up the "dev console":https://ably.com/accounts/any/apps/any/console of your first app before instantiating your client so that you can see what happens.
+
+* Run the following function to instantiate the SDK and establish a connection to Ably. At the minimum you need to provide an authentication mechanism. Use an API key for simplicity, but you should use token authentication in a production app. A @clientId@ ensures the client is identified, which is required to use certain features, such as presence:
+
+```[javascript]
+const PubSub = require('ably');
```

Comment:
Indentation is a tad off.

```const PubSub = require('ably');

async function getStarted() {
  const realtimeClient = new PubSub.Realtime({ key: '{{API_KEY}}', clientId: 'my-first-client' });

  await realtimeClient.connection.once('connected');
  console.log(`Made my first connection!`);
};

getStarted();```

Reply:
Good spot... not sure what was going on with those: 0a9d8372589232a3ad33b6109ed5496870cb258c